### PR TITLE
Version 1.40.1 → 1.40.2

### PR DESCRIPTION
**Pull request by Gabriel439**
Fixes https://github.com/dhall-lang/dhall-haskell/issues/2339

### CODE REVIEW

File: dhall-csv/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.0.1
+
+* [Build against `tasty-silver` 3.3](https://github.com/dhall-lang/dhall-haskell/pull/2307)
```
Comment (sjakobi) [inline dhall-csv/CHANGELOG.md:3]: I'm not sure this is worth listing: You've already published this change as a revision: https://hackage.haskell.org/package/dhall-csv-1.0.0/revisions/

File: dhall/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
+1.40.2
+
+* [Supports standard version 21.1.0](https://github.com/dhall-lang/dhall-haskell/pull/2236)
+    * [The interpreter now supports customizing headers using a `headers.dhall`
+      configuration file](https://github.com/dhall-lang/dhall-haskell/pull/2236)
+    * [Generalize grammar for certain annotations](https://github.com/dhall-lang/dhall-haskell/pull/2320)
+    * [Add support for a trailing line comment without a newline](https://github.com/dhall-lang/dhall-haskell/pull/2309)
```
Comment (sjakobi) [inline dhall/CHANGELOG.md:7]: It would be good to update the `dhall-lang` submodule to reflect this support.

**Review by sjakobi — APPROVED** [review]
Some of the dependency updates aren't worth listing IMHO, particularly if they've already been released as Hackage revisions. But I guess they don't hurt much either.

File: dhall-csv/CHANGELOG.md (hunk 1 above)
Comment (Gabriel439) [reply dhall-csv/CHANGELOG.md:3]: Usually I mention these sorts of changes if there is nothing else to mention, mainly to explain why we're publishing a new release at all

**Pull request merged**